package.json

```diff
@@ -236,7 +236,8 @@
     }
   },
   "overrides": {
-    "resolve": "1.20.0"
+    "resolve": "1.20.0",
+    "breakword": "^1.0.6"
```

Comment:
probably you need to update changeset-cli itself as this library is using that directly, instead of overriding